### PR TITLE
WSTEAM1-472: Update cookie links on Canonical and AMP cookie banner.

### PR DESCRIPTION
**Pull request by shayneahchoon**
Resolves JIRA [WSTEAM1-472](https://jira.dev.bbc.co.uk/browse/WSTEAM1-472)

Overall changes
======
Changes .co.uk to .com  for WS cookie links. This change does NOT affect: 
- WS Cymrufyw
- WS Naidheachdan
- WS News
- WS Newsround
- WS Scotland
- WS Sport

Testing
======
1. Check cookie links redirect to .com on non UK sites (as outlined by red arrows): 
Screenshot 2023-07-28 at 13 51 39
Screenshot 2023-07-28 at 13 57 54
Screenshot 2023-07-28 at 13 58 03

### CODE REVIEW

File: src/app/lib/config/services/gujarati.ts [hunk 1]
```diff
@@ -157,7 +157,7 @@ export const service: DefaultServiceConfig = {
                   'અમારી વેબસાઇટને યોગ્ય રીતે કાર્યરત રાખવા માટે સીમિત સ્તરે કેટલીક જાણકારીઓ સેવ કરવી પડે છે જેના માટે અમે આપની મંજૂરી નથી માગતા.',
                 para4: {
                   text: 'એ જરૂરી જાણકારીઓ વિશે વધારે વાંચો જે અમે આપની ડિવાઇસ પર સ્ટોર કર્યું છે જેથી અમારું પેજ આપને યોગ્ય રીતે દેખાય.',
-                  url: 'https://www.bbc.co.uk/usingthebbc/strictly-necessary-cookies/',
+                  url: 'https://www.bbc.com/usingthebbc/strictly-necessary-cookies/',
```
Comment (amoore108) [inline src/app/lib/config/services/gujarati.ts]: These appear to 404 for me. Although the previous `.co.uk` one 404s also 🤔

Comment (karinathomasbbc) [reply src/app/lib/config/services/gujarati.ts]: This one appears to work: https://www.bbc.com/usingthebbc/cookies/strictly-necessary-cookies/

Comment (shayneahchoon) [reply src/app/lib/config/services/gujarati.ts]: Will take a look, thanks 👍

Comment (shayneahchoon) [reply src/app/lib/config/services/gujarati.ts]: Heya, just checked, it seems to work on my localhost.

Comment (shayneahchoon) [reply src/app/lib/config/services/gujarati.ts]: Heya, soz I was looking at the wrong paragraph link, I've changed it to the correct one now :)

Comment (karinathomasbbc) [reply src/app/lib/config/services/gujarati.ts]: Cool. I think some of the snapshots might need updating, but looks good to me!

**Review by karinathomasbbc — APPROVED** [review]

**Review by HarveyPeachey — APPROVED** [review]

**Pull request merged**